src/AWS.Deploy.CLI/Commands/TypeHints/VPCConnectorCommand.cs

```diff
@@ -76,6 +76,30 @@ public async Task<object> Execute(Recommendation recommendation, OptionSettingIt
             _toolInteractiveService.WriteLine();
             if (createNewVPCConnector)
             {
+                var availableVpcs = await _awsResourceQueryer.GetListOfVpcs();
+                if (!availableVpcs.Any())
+                {
+                    _toolInteractiveService.WriteLine("Your account does not contain a VPC, so we will create one for you and assign it's Subnets and Security Groups to the VPC Connector.");
+                    return new VPCConnectorTypeHintResponse
+                    {
+                        UseVPCConnector = true,
+                        CreateNew = true,
+                        CreateNewVpc = true
+                    };
+                }
```

Comment:
Similar comment to Beanstalk above, how can we apply this messaging/behavior to server mode?

Reply:
Done. Toolkit may not show validations correctly, but I have validated they work in server mode. Working with Chris to make sure this change works in the toolkit.